Compiler error with custom yaml-cpp path

```
[ 83%] Building CXX object src/tools/tools/CMakeFiles/halley-tools.dir/src/assets/asset_importer.cpp.o
In file included from /home/simon/code/halley/src/tools/tools/src/assets/asset_importer.cpp:8:0:
/home/simon/code/halley/src/tools/tools/src/assets/importers/material_importer.h:3:32: fatal error: yaml-cpp/node/node.h: No such file or directory
 #include <yaml-cpp/node/node.h>
                                ^
compilation terminated.
```

Looking at https://github.com/amzeratul/halley/blob/master/src/tools/tools/CMakeLists.txt it looks like `${YAMLCPP_INCLUDE_DIR}` is in the wrong place, in `add_library` instead of in `include_directories`.

--- a/src/tools/tools/CMakeLists.txt
+++ b/src/tools/tools/CMakeLists.txt
@@ -1,6 +1,6 @@
 project (halley-tools)
 
-include_directories(${BOOST_INCLUDE_DIR} ${FREETYPE_INCLUDE_DIRS} "include" "../../engine/core/include" "../../engine/utils/include" "../../engine/audio/include" "../../engine/net/include" "../../contrib/libogg/include" "../../contrib/libvorbis/include")
+include_directories(${BOOST_INCLUDE_DIR} ${FREETYPE_INCLUDE_DIRS} ${YAMLCPP_INCLUDE_DIR} "include" "../../engine/core/include" "../../engine/utils/include" "../../engine/audio/include" "../../engine/net/include" "../../contrib/libogg/include" "../../contrib/libvorbis/include")
 
 set(SOURCES
 
@@ -141,7 +141,7 @@ set(HEADERS
 assign_source_group(${SOURCES})
 assign_source_group(${HEADERS})
 
-add_library (halley-tools ${SOURCES} ${HEADERS} ${YAMLCPP_INCLUDE_DIR})
+add_library (halley-tools ${SOURCES} ${HEADERS})
 
 target_link_libraries (halley-tools
     halley-core

